Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0`

Reply:
updated